editor/modes/visual-editor/block-list.js

```diff
@@ -248,18 +268,24 @@ class VisualEditorBlockList extends Component {
 						onKeyDown={ noop }
 					/>
 				}
-				<div className="editor-visual-editor__continue-writing">
+				<div
+					className={ continueWritingClassname }
+					onFocus={ this.onContinueWritingFocus }
```

Comment:
Minor: While it has the downside of not being a shared function reference (bypassing benefit if we ever adopted [PureComponent](https://facebook.github.io/react/docs/react-api.html#react.purecomponent)), we could consolidate `onFocus` and `onBlur` toggle to a single helper function:

```js
toggleContinueWritingControls( showContinueWritingControls ) {
	return () => this.setState( { showContinueWritingControls } );
}

// ...

<div
	className={ continueWritingClassname }
	onFocus={ this.toggleContinueWritingControls( true ) }
	onBlur={ this.toggleContinueWritingControls( false ) }
>
```